cmd/genji/dump.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/genjidb/genji/engine/badgerengine"
+	"github.com/genjidb/genji/engine/boltengine"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/genjidb/genji/engine"
+
+	"github.com/genjidb/genji"
+	"github.com/genjidb/genji/cmd/genji/shell"
+)
+
+func executeDump(ctx context.Context, f string, tables []string, e string, dbPath string, w io.Writer) error {
```

Comment:
I don't think we need to pass `os.Stdout` as a parameter here.

```suggestion
func executeDump(ctx context.Context, f string, tables []string, e string, dbPath string) error {
```

Reply:
I kept the same logic as the `RunDumpCommand` it is better for testing... And the same argument that above.

Reply:
I suggest to have something like this so that you can easily test
```go
func executeDump(ctx context.Context, w io.Writer, tables []string, e, dbPath string) error
```